spec/models/room_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Room, type: :model do
+  describe 'basic validations' do
+    it { is_expected.to validate_presence_of(:number) }
+    it { is_expected.to validate_presence_of(:beds) }
+    it { is_expected.not_to allow_value(-1).for(:beds) }
+    it { is_expected.to belong_to(:suite) }
+    describe 'number uniqueness' do
```

Comment:
If it is easy and simple, I believe that `shoulda_matchers` offers a way to do this and we should switch.